webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableFormats.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export const hostsCountFormat = (value, { rowData: { name } }) => (
+  <a href={`hosts?search=model+%3D+"${encodeURI(name)}"`}>{value}</a>
+);
+
+export const actionsFormat = (value, { rowData: { name, id } }) =>
+  (value && value.includes('delete') ? (
+    <span className="btn btn-sm btn-default">
+      <a
+        data-confirm={__(`Delete ${name}`)}
+        data-id={`aid_models_${id}-${encodeURI(name)}`}
+        rel="nofollow"
+        data-method="delete"
+        href={`/models/${id}-${encodeURI(name)}`}>
```

Comment:
nitpick: please ensure you are using prettier, on my env it format this (and others) differently.

Reply:
Can you show me how it looks on your env after running `prettier`. Thanks.